spec/lib/sidekiq_unique_jobs/unique_args_spec.rb

```diff
@@ -4,6 +4,44 @@
   let(:item) { { 'class' => 'UntilExecutedJob', 'queue' => 'myqueue', 'args' => [[1, 2]] } }
   subject { described_class.new(item) }
 
+  context '#unique_digest' do
```

Comment:
I find it more intention revealing to use `describe '#unique_digest'` for when speccing methods. The whole things should preferably look more like:

```ruby
describe '#unique_digest' do
    let(:item) { item_options.merge('args' => [1, 2, 'type' => 'it'] ) }

    shared_examples 'unique digest' do
      context 'given another item' do
        let(:another_subject) { described_class.new(another_item) }

        context 'with the same unique args' do
          let(:another_item) { item_options.merge('args' => [1, 2, 'type' => 'it']) }
          it 'equals to unique_digest for that item' do
            expect(subject.unique_digest).to eq(another_subject.unique_digest)
          end
        end

        context 'with different unique args' do
          let(:another_item) { item_options.merge('args' => [1, 3, 'type' => 'that']) }
          it 'differs from unique_digest for that item' do
            expect(subject.unique_digest).to_not eq(another_subject.unique_digest)
          end
        end
      end
    end

    context 'when unique_args is a proc' do
      let(:item_options) do
        { 'class' => 'UntilExecutedJob', 'queue' => 'myqueue',
          'unique_args' => Proc.new { |args| args[1] }}
      end
      it_behaves_like 'unique digest'
    end

    context 'when unique_args is a symbol' do
      let(:item_options) do
        { 'class' => 'UniqueJobWithFilterMethod', 'queue' => 'myqueue',
          'unique_args' => :filtered_args }
      end
      it_behaves_like 'unique digest'
    end
  end
```